### ISSUE TITLE
Make lotus install on JRuby

### ISSUE THREAD

**Issue by PragTob** [lotus/lotus#146]
Hi there,

right now the install fails on JRuby as the `sqlite3` gem needs native extensions.

When I got time, I'll fix this - should be as easy as using [`jdbc-sqlite3`](https://github.com/jruby/activerecord-jdbc-adapter/tree/master/jdbc-sqlite3)

```ruby
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    /home/tobi/.rvm/rubies/jruby-head/bin/jruby -r ./siteconf20150203-4310-1vbkw0d.rb extconf.rb 
NotImplementedError: C extensions are not supported
      (root) at /home/tobi/.rvm/rubies/jruby-head/lib/ruby/stdlib/mkmf.rb:1
     require at org/jruby/RubyKernel.java:964
      (root) at /home/tobi/.rvm/rubies/jruby-head/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1
  __script__ at extconf.rb:3

extconf failed, exit code 1

Gem files will remain installed in /home/tobi/.rvm/gems/jruby-head@lotus/gems/sqlite3-1.3.10 for inspection.
Results logged to /home/tobi/.rvm/gems/jruby-head@lotus/extensions/universal-java-1.7/2.2/sqlite3-1.3.10/gem_make.out
An error occurred while installing sqlite3 (1.3.10), and Bundler cannot
continue.
Make sure that `gem install sqlite3 -v '1.3.10'` succeeds before bundling.

```

**Comment by jodosha** [comment]
Closed by #147

**Comment by PragTob** [comment]
I'm closing this :)
